packages/server/postgres/queries/insertStripeQuantityMismatchLogging.ts

```diff
@@ -0,0 +1,35 @@
+import getPg from '../getPg'
+import OrganizationUser from '../../database/types/OrganizationUser'
+import {insertStripeQuantityMismatchLoggingQuery} from './generated/insertStripeQuantityMismatchLoggingQuery'
+
+const insertStripeQuantityMismatchLogging = async (
+  userId: string,
+  eventTime: Date,
+  eventType: string,
+  stripePreviousQuantity: number,
+  stripeNextQuantity: number,
+  orgUsers: OrganizationUser[]
+) => {
+  return insertStripeQuantityMismatchLoggingQuery.run(
+    {
+      userId,
+      eventTime,
+      eventType,
+      stripePreviousQuantity,
+      stripeNextQuantity,
+      orgUsers: orgUsers.map(({id, inactive, joinedAt, removedAt, userId, role, tier}) => {
+        return {
+          id,
+          inactive,
+          joinedAt: joinedAt.getTime(),
+          removedAt: removedAt?.getTime() ?? -1,
```

Comment:
instead of `getTime()` could we use `toJSON`? that'll give us an ISO8601 date string which will still be sortable, but also human readable.

the fallback value can just be an empty string